Apps/Sandcastle/gallery/Cesium OSM Buildings Styling.html

```diff
@@ -0,0 +1,260 @@
+            defines: {
+              material: "${feature['building:material']}",
+            },
+            color: {
+              conditions: [
+                ["${material} === null", "color('white')"],
+                ["${material} === undefined", "color('white')"],
+                ["${material} === 'glass'", "color('skyblue', 0.5)"],
+                ["${material} === 'concrete'", "color('grey')"],
+                ["${material} === 'brick'", "color('indianred')"],
+                ["${material} === 'stone'", "color('lightslategrey')"],
+                ["${material} === 'metal'", "color('lightgrey')"],
+                ["${material} === 'steel'", "color('lightsteelblue')"],
+                ["true", "color('white')"],
+              ],
+            },
+          });
+        }
+
+        function colorByType() {
+          buildings.style = new Cesium.Cesium3DTileStyle({
+            color: {
+              conditions: [
+                [
+                  "${feature['building']} === 'apartments' || ${feature['building']} === 'residential'",
+                  "color('cyan', 0.9)",
+                ],
+                [
+                  "${feature['building']} === 'school' || ${feature['building']} === 'university'",
+                  "color('lawngreen', 0.9)",
+                ],
+                [
+                  "${feature['building']} === 'hospital'",
+                  "color('yellow', 0.9)",
+                ],
+                [
+                  "${feature['building']} === 'construction'",
+                  "color('orangered', 0.9)",
+                ],
+                [
+                  "${feature['building']} === 'office' || ${feature['building']} === 'commercial'",
+                  "color('blueviolet', 0.9)",
+                ],
+                [true, "color('white')"],
+              ],
+            },
+          });
+        }
+
+        function colorByDistance() {
+          buildings.style = new Cesium.Cesium3DTileStyle({
+            defines: {
+              distance:
+                "distance(vec2(radians(${feature['cesium#longitude']}), radians(${feature['cesium#latitude']})), vec2(-2.13539829704, 0.83113469098))",
+            },
+            color: {
+              conditions: [
+                ["${distance} > 0.0010", "color('gray')"],
+                [
+                  "${distance} > 0.0006",
+                  "mix(color('yellow'), color('red'), (${distance} - 0.008) / 0.0004)",
+                ],
+                [
+                  "${distance} > 0.0002",
+                  "mix(color('green'), color('yellow'), (${distance} - 0.0004) / 0.0004)",
+                ],
+                ["${distance} < 0.00001", "color('white')"],
+                [
+                  "true",
+                  "mix(color('blue'), color('green'), ${distance} / 0.0004)",
+                ],
+              ],
+            },
+          });
+        }
```

Comment:
Can we simplify this by:

* Using degrees instead of radians in the distance check at the top? 
* It would also be nice to explain in a comment where you got that lat/lon from. The easiest thing is to click on a building and take the lat/lon from that, but not everyone may know that.
* Instead of using mix can we just have distinct colors for each? And maybe have them be, red, yellow, green.
* Call this "Color by distance to Space Needle" so it's obvious "distance to what"